sshd_config.5

```diff
@@ -1595,6 +1595,11 @@ Specifies how long to refuse clients that cause a crash of
 .It Cm authfail:duration
 Specifies how long to refuse clients that disconnect after making one or more
 unsuccessful authentication attempts (default: 5s).
+.It Cm admin-refused:duration
```

Comment:
I would call this connection-refused instead or refused-connection

the admin-refused makes it harder for me to link this to the RefuseConnection keyword.

Reply:
Good idea - I went a step further and renamed it to `refuseconnection` :)